sbi/simulators/simutils.py

```diff
@@ -134,6 +135,79 @@ def simulation_wrapper(
     return parameters, observations
 
 
+def simulation_wrapper_batch(
+        simulator: Callable,
+        parameter_sample_fn: Callable,
+        num_samples: int,
+        simulation_batch_size: int):
+    """
+    Runs simulations in a loop. Each loop simulates simulation_batch_size parameter sets until
+    a total of num_samples is reached
+
+    Args:
+        simulator: Simulator function.
+            If simulation_batch_size == 1: takes in thetas of shape (num_dim_theta)
```

Comment:
What are thetas? does the user know at this point? I don't see thetas in the signature.